langlib/lang.array/src/main/ballerina/src/lang.array/Module.md

```diff
@@ -1,3 +1,5 @@
 ## Module Overview
 
-This module provides lang library list operations defined by the language specification 2019R2.
+This module provides lang library list operations defined by the language specification 2019R3. All the functions listed 
+below are supported by resizable arrays. Fixed-length arrays and tuples do not support any operations that violate the 
+properties of these types. e.g., pushing a value to a tuple. 
```

Comment:
```suggestion
properties of these types. e.g., pushing a value to a fixed-length array.
```

Not sure the tuple example is correct. Even if we can't now, this should be possible for a tuple with a compatible rest field right?

Reply:
Yes, Since this is module MD, I think simply saying 2019R3 is enough for now. We can improve the doc later.

Reply:
Wrote it for the current functionality we have.